src/platform_impl/macos/window.rs

```diff
@@ -344,6 +356,13 @@ impl WinitWindow {
                     }
                 }
 
+                if let Parent::ChildOf(parent) = pl_attrs.parent {
+                    #[allow(clippy::let_unit_value)]
+                    unsafe {
+                        let () = msg_send![parent as *mut Object, addChildWindow: &*this, ordered: NSWindowOrderingMode::NSWindowAbove];
+                    }
+                }
+
```

Comment:
Is there a specific reason you placed this code here instead of further down? As-in, does the `addChildWindow:ordered:` call need to happen at a certain point, or can it be done basically anytime?

Reply:
It can be further down but I saw every platform specific attributes are handled here.
Where do you think is better?

Reply:
No then it's fine, just wanted to know ;)